ext/tellafriend/tellafriend.php

```diff
@@ -54,7 +54,7 @@ function tellafriend_civicrm_tabset($tabsetName, &$tabs, $context) {
       'template' => FALSE,
       'count' => FALSE,
       'icon' => FALSE,
-      'url' => $tabsetName === 'civicrm/event/manage' ? 'civicrm/event/manage/friend' : '',
+      'url' => $tabsetName === '/civicrm/event/manage' ? '/civicrm/event/manage/friend' : '',
```

Comment:
@eileenmcnaughton the first slash doesn't make sense. It's not possible because of the outer contition that ensures it *doesn't* start with a slash.

Reply:
@colemanw huh - and yet it works - probably because it does a good job if empay

Reply:
OK - I can see where there is a conflict between 2 paths....

Reply:
It seems the code was needed for this access but broke the other  - works for both now

![image](https://github.com/user-attachments/assets/807f13f5-ca9c-4c7f-84a1-3b1ddac98ef3)